code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -880,19 +880,20 @@ GLOBAL_LIST_EMPTY(PDAs)
 	else
 		remove_pen()
 
-/obj/item/pda/CtrlClick()
+/obj/item/pda/CtrlClick(mob/user)
 	..()
 
 	if(isturf(loc)) //stops the user from dragging the PDA by ctrl-clicking it.
 		return
 
-	remove_pen()
+	remove_pen(user)
 
 /obj/item/pda/verb/verb_toggle_light()
```

Comment:
rename toggle_light to something else, and then verb_toggle_light to toggle_light, so you don't need the special snowflake code on the action to handle the pda toggle.

Reply:
Done.